<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Q PSN - 0.2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>891</width>
      <height>551</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Content</string>
     </attribute>
     <widget class="QTableView" name="tableViewContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>671</width>
        <height>481</height>
       </rect>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="labelUploadedBy">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>400</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>labelUB</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDownload">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>420</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRap">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>440</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Download RAP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLinkCopy">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>460</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy link</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSearch">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>70</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelContentSearch">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>50</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Search for content:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSearchResult">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>90</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>labelSearchResult</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="pteDescription">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>110</y>
        <width>201</width>
        <height>281</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelFilter">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>0</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbFilter">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PSP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PSVita</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PS1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PS2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PS3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PSN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DLC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Demo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mini</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EDAT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Theme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wallpaper</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Avatar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Video</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="labelTotalEntries">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Submit</string>
     </attribute>
     <widget class="QLabel" name="lSubGameID">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Game ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lSubTitle">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Title:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lSubType">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lSubRegion">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Region:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lSubLink">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PSN URL:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lSubRapName">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RAP name (content ID):</string>
      </property>
     </widget>
     <widget class="QLabel" name="lSubRapData">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RAP data (hex):</string>
      </property>
     </widget>
     <widget class="QLabel" name="lSubDescription">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Small description:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSubGameID">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>9</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSubTitle">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSubLink">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>180</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSubRapName">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>40</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSubRapData">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maxLength">
       <number>32</number>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="pteSubDescription">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>300</y>
        <width>381</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbSubType">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>381</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>PSP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PSVita</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PS1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PS2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PS3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PSN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DLC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Demo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mini</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EDAT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Theme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wallpaper</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Avatar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Video</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbSubRegion">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>381</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>EU</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>US</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HK</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnSubSubmit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>410</y>
        <width>241</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="lSubUploader">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Posted by:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leUploaderName">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>380</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>News</string>
     </attribute>
     <widget class="QPlainTextEdit" name="pteNews">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>811</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Informations</string>
     </attribute>
     <widget class="QPlainTextEdit" name="pteLicense">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>581</width>
        <height>231</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Copyright (C) 2014,2015 - sguerrini97

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program; if not, write to the Free Software Foundation,
Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSrc">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>420</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Source code</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelLogo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>551</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res/res.qrc">:/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
